omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_postgres_validator.rb

```diff
@@ -20,8 +20,11 @@ class PostgresqlPreflightValidator < PreflightValidator
   # we ship. When we bumped the version we ship to 9.6, we haven't yet
   # introduced any changes that _require_ 9.6. So, these constants reflect the
   # actually required PG version.
-  REQUIRED_MAJOR = 9
-  REQUIRED_MINOR = 2
+  REQUIRED_MAJOR  = 9
+  REQUIRED_MINOR  = 2
+
+  # supported PG versions (not including required versions)
+  SUPPORTED_MAJOR = [12, 13]
```

Comment:
Do we want to support 12 since we're not testing it anywhere?

Reply:
Should change the error to a warning? If we do that then we can remove 12 from the array.

Reply:
I removed it from the array.